halyard-web/lib/systemd/system/halyard.service

```diff
@@ -9,4 +9,4 @@ ExecStart=/opt/halyard/bin/halyard
 User=spinnaker
 
 [Install]
-WantedBy=user.target
+WantedBy=multi-user.target
```

Comment:
Resolves this issue:
```
Unit /lib/systemd/system/halyard.service is added as a dependency to a non-existent unit user.target.
```
Which also prevents the service from starting automatically on reboot.